docker-compose.dev.yml

```diff
@@ -0,0 +1,6 @@
+version: '2'
+services:
+  webapp:
+    entrypoint: /bin/bash -c "chmod +x /webodm/*.sh && /webodm/wait-for-it.sh db:5432 -- /webodm/start.sh --create-default-pnode --setup-devenv"
```

Comment:
Is it possible to skip the `chmod` by ensuring that files have appropriate permissions in the git repo?